OpenRA.Mods.Common/Traits/Sound/DeathSounds.cs

```diff
@@ -26,6 +26,12 @@ public class DeathSoundsInfo : ITraitInfo
 		public readonly string[] DeathTypes = { };
 
 		public object Create(ActorInitializer init) { return new DeathSounds(this); }
+
+		public void CheckBogusYaml(string actorName, string traitName)
+		{
+			if (VolumeMultiplier < 0)
+				throw new YamlException("{0}.{1}.VolumeMultiplier is negative.".F(actorName, traitName));
```

Comment:
`YamlException` is used for more low-level problems in the parsing. Maybe use `InvalidArgumentException` instead.